### PR TITLE
feat(components/popovers): remove string as valid input on dropdown buttonType

### PR DESCRIPTION
**Pull request by Blackbaud-CoreyArcher**
[AB#2331872](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2331872)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1155/storybook/)

Component Storybooks:

- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1155/storybooks/ag-grid-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1155/storybooks/layout-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/1155/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1155/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1155/storybooks/tabs-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1155/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1155/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1155/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1155/playground/)

File: libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts [hunk 1]
```diff
@@ -212,52 +202,22 @@ export class SkyDropdownComponent implements OnInit, OnDestroy {
...
-
-  #_buttonStyle = 'default';
-
-  #_buttonType: SkyDropdownButtonType | string = DEFAULT_BUTTON_TYPE;
-
+  #ngUnsubscribe = new Subject<void>();
+  #overlayService = inject(SkyOverlayService);
+  #positionTimeout: number | undefined;
+  #themeSvc = inject(SkyThemeService, { optional: true });
+  #zIndex = inject(SKY_STACKING_CONTEXT, { optional: true })?.zIndex;
```
Comment (Blackbaud-SteveBrush) [inline libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts:213]: Let's group the `inject` properties at the bottom of the list, nearest to the `constructor`.

**Review by Blackbaud-SteveBrush — COMMENTED** [review]
Remember to put a `BREAKING CHANGE` comment when this PR is merged.

File: libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts (hunk 1 above)
Comment (Blackbaud-CoreyArcher) [reply libs/components/popovers/src/lib/modules/dropdown/dropdown.component.ts:213]: updated

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**